TECHNICAL_MANIFEST.md

```
@@ -0,0 +1,1618 @@
+CREATE INDEX ON gold.bpl_embeddings USING GIN (metadata jsonb_path_ops);
+```
+
+---
+
+## RAG Components
+
+### Component Architecture (Modular Design)
+
+The RAG system is organized into independent, testable modules under `current_fall2025/scripts/RAG/`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)